Firmware/MVP/gridCharger.cpp

```diff
@@ -135,7 +106,7 @@ void runFansIfNeeded(void)
 
 void powered_handler(void)
 {
-    static bool gridChargerEnabled_previous = NO;
+    static uint8_t chargerState_previous = NO__UNINITIALIZED;
```

Comment:
Conceptually this works, but I don't like reusing the gridCharger.h #define constants here.  The grid charger code is the most safety critical aspect of LiBCM, and I prefer the old boolean variable here inside this function.

Reply:
If we do keep your proposed #define changes, we need to change your new uint8_t variable name to better explain what's going on... "chargerState_previous = NO__UNINITIALIZED" reads better as "chargerEnabled_previous = NO_UNITIALIZED"

Reply:
After more thought, I ultimately do like the change from bool to uint8_t, but will change the variable name to "chargingAllowed_previous" after merging this PR.